buildSrc/build.gradle.kts

```diff
@@ -10,6 +10,8 @@ plugins {
 
 val cacheRedirectorEnabled = System.getenv("CACHE_REDIRECTOR")?.toBoolean() == true
 val buildSnapshotTrain = properties["build_snapshot_train"]?.toString()?.toBoolean() == true
+val kotlinDevUrl = project.rootProject.properties["kotlin_repo_url"] as? String // WA for CacheRedirector.kt
```

Comment:
In CommunityProjectsBuild, if this property is not set, a system environment variable is queried. Why not here? If intentional, please add a comment.